hyundai_kia_connect_api/Vehicle.py

```diff
@@ -74,6 +74,8 @@ class Vehicle:
     year: int = None
     VIN: str = None
     key: str = None
+    protocolType: int = None
```

Comment:
Can you follow current naming convention?

Reply:
Do you mean instead of Camel Case notation I should use underscore? Good point, I will fix it.

Reply:
If protocol type isn't used in an if statement maybe we don't record it?

Reply:
I fixed the naming and removed the attribute as proposed above. Additionaly, I continude with the mapping of the attributes.